packages/contracts/docs/osx/01-how-it-works/02-framework/02-plugin-management/01-plugin-repo/01-plugin-repo-creation.md

```diff
@@ -40,7 +40,7 @@ function createPluginRepoWithFirstVersion(
 ) external returns (PluginRepo pluginRepo);
 ```
 
-which creates a `PluginRepo` with the first version `1.0` inside and registers it in the Aragon OSx `PluginRepoRegistry` contract with an [ENS subdomain](../../03-ens-names.md) under the `dao-plugin.eth` domain managed by Aragon.
+which creates a `PluginRepo` with the first version `1.0` inside and registers it in the Aragon OSx `PluginRepoRegistry` contract with an [ENS subdomain](../../03-ens-names.md) under the `plugin.dao.eth` domain managed by Aragon.
```

Comment:
It creates with `1.1` as the first version. maybe it's a good idea to mention `release/build` here.

Reply:
Fixed by 35bc0711708f5566ee8c696093adab9c6a69a61e